docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: docs
+title: Why TestCafe?
+permalink: /documentation/why-testcafe/
+---
+
+# Why TestCafe?
+
+TestCafe excels where other end-to-end testing solutions fall short.
+
+## One NPM package
+
+TestCafe doesn't require the use of extra tools and frameworks. All you need to get up and running is a recent version of Node.js, a browser — any browser, and the TestCafe NPM package.
+
+## As simple and extendable as JavaScript itself
+
+TestCafe tests are full-fledged Node.js apps. They are compatible with your favourite JavaScript libraries and preprocessors.
+
+## Complex tests made easy
+
+TestCafe’s intuitive and powerful toolkit makes testing easy, whether you’re working with a landing page or a complex web app.
+
+### Mock Requests
```

Comment:
Does this continue the "simple" message? (Probably not). It's more likely part of the "Powerful and Extendable" message. So again - there should be some separation between the two.

Reply:
This section is about complex things made easy, mocking is one of these things. Does it seem more appropriate to you if we emphasize ease-of-use here as well?